docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -655,6 +659,50 @@ You can specify multiple selectors by adding elements to the list.
   Once `databaseResources` is set, you can dynamically register database with `tsh` by following [this guide](../../database-access/guides/dynamic-registration.mdx).
 </Admonition>
 
+
+## kubernetesDiscovery
+
+| Type   | Default value | Required?                                                                                                                             |
+|--------|---------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `list` | `See below`   | When the `discovery` chart role is used, `kubernetesDiscovery` is required. |
+
+Default value:
+
+  ```yaml
+  kubernetesDiscovery:
+  - types: ["app"]
+    namespaces: ["*"]
+    labels:
+      "*": "*"
+  ```
+
+`kubernetesDiscovery` is a YAML list object detailing the filters for the services of the Kubernetes cluster that should be discovered
+ and exposed as apps in Teleport. By default the Discovery Service will try to find all apps that are serving HTTP/HTTPS, but you can manually
+ make it expose TCP apps as well, using Kubernetes annotations.
+
+You can specify multiple filters, each of which can include these parameters:
+- `types` is a list of types of services to be exposed in Teleport, currently only "app" value is supported, with "db" value planned.
```

Comment:
```suggestion
- `types` is a list of types of services to be exposed in Teleport. Currently only `app` is supported.
```
Grammar tweak. Also removing promises, which I don't think will help a reader looking for current documentation.